### ISSUE TITLE
Positive feedback in review form

### ISSUE THREAD

**Issue by yoe** [yoe/sreview#46]
When the use clicks on any one of the buttons in the review form, nothing apparently happens. This is confusing.

The review form should produce some form of positive feedback that "start" or "end" marker has been set.

This may possibly depend on #45 and/or #44.

**Comment by yoe** [comment]
this is a duplicate of #31, which was closed (I'm an idiot)
